ios/InfoPlist_Bebbo/ro-MD.lproj/InfoPlist.strings

```diff
@@ -0,0 +1,8 @@
+/* 
+  InfoPlist.strings
+  ParentBuddyApp
+
+  Created by Kalyani Kaushik on 18/11/21.
```

Comment:
Are you adding a new file @ansariDataamanulla?
Should we update the author?

Reply:
yes based on choose localization